common/djangoapps/third_party_auth/api/urls.py

```diff
@@ -3,13 +3,14 @@
 from django.conf.urls import patterns, url
 
 from .views import UserView, UserMappingView
+from django.conf import settings
+
 
-USERNAME_PATTERN = r'(?P<username>[\w.+-]+)'
 PROVIDER_PATTERN = r'(?P<provider_id>[\w.+-]+)(?:\:(?P<idp_slug>[\w.+-]+))?'
 
 urlpatterns = patterns(
     '',
-    url(r'^v0/users/' + USERNAME_PATTERN + '$', UserView.as_view(), name='third_party_auth_users_api'),
+    url(r'^v0/users/' + settings.USERNAME_PATTERN + '$', UserView.as_view(), name='third_party_auth_users_api'),
```

Comment:
I agree with you, but trying to consistent with the rest of the file here. Unless it is absolutely required for this PR to get merged, I prefer not to touch the other lines.